src/types.ts

```diff
@@ -1,4 +1,4 @@
-import type AggregateError from "aggregate-error";
+import AggregateError = require("aggregate-error");
```

Comment:
Why not use the syntax recommended by the test action?
`import * as ns from "mod"`

Reply:
Trying this approach I get the following error:
![image](https://user-images.githubusercontent.com/2574275/92251311-f044c800-eecc-11ea-89db-3ad4dd3c013a.png)

Also, trying `import AggregateError from ...` seems to be an issue:
![image](https://user-images.githubusercontent.com/2574275/92251223-cc818200-eecc-11ea-9efb-d0a6a4cc405f.png)

---

The reason I was using `import AggregateError = require()` is because what I found in Typescript docs: https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require but not sure how to handle this.

* https://github.com/sindresorhus/aggregate-error/blob/d5bb4ac02a43f005ec7ad45f6e62919d7ebed0e5/index.d.ts#L51

---

Any ideas @wolfy1339